City-of-Helsinki/drupal-hdbt
# UHF-8622: LoadingSpinner accessibility fixes

# [UHF-8622](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8622)


Loading spinner's status messages interference screen readers with unnecessary information.

## What was done


* Removed status messages from loading spinners
* Some code cleanup

## How to install
* You need to have the following instances
  * https://github.com/City-of-Helsinki/drupal-helfi-kymp
  * https://github.com/City-of-Helsinki/drupal-helfi-rekry
  * https://github.com/City-of-Helsinki/drupal-helfi-tyo-yrittaminen
  * https://github.com/City-of-Helsinki/drupal-helfi-etusivu
  * https://github.com/City-of-Helsinki/drupal-helfi-kasvatus-koulutus
* Make sure each instance is up and running on latest dev branch.
  * `git pull origin dev`
  * `make fresh`
* Update the HDBT theme on each
  * `composer require drupal/hdbt:dev-UHF-8622_LoadingSpinner_accessibility`
* Run `make drush-cr`

## How to test


* [x] Check that loading spinner doesn't have status messages. 
See contents of `<div>` with id `hds-loading-spinner-notification-area`. It's also preferable to slow down your browser's speed to catch this div's existence.
  * [x] District and Project search
https://helfi-kymp.docker.so/fi/kaupunkiymparisto-ja-liikenne/kaupunkisuunnittelu-ja-rakentaminen/hae-suunnitelmia-ja-hankkeita
  * [x] Job search
https://helfi-rekry.docker.so/fi/avoimet-tyopaikat/etsi-avoimia-tyopaikkoja
  * [x] Linkedevents
https://helfi-elo.docker.so/fi/yritykset-ja-tyo/tapahtumat
  * [x] News Archive
https://helfi-etusivu.docker.so/fi/uutiset
  * [x] School search
https://helfi-kasko.docker.so/fi/kasvatus-ja-koulutus
You have to create a Landing Page with School Search paragraph

## Designers review





[UHF-8622]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8622?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
